google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/WriteChannelConfiguration.java

```diff
@@ -159,6 +165,13 @@ public Builder setFormatOptions(FormatOptions formatOptions) {
 
 
     @Override
+    public Builder setNullMarker(String nullMarker) {
+      this.nullMarker = nullMarker;
+      return this;
+    }
+
+
```

Comment:
These extra lines currently appear in hundreds of files. Do you think we should do a broad reformatting change?